com.rlabrecque.steamworks.net/Runtime/Steam.cs

```diff
@@ -111,7 +114,42 @@ public static ESteamAPIInitResult InitEx(out string OutSteamErrMsg)
 			}
 		}
 
+#if STEAMWORKS_ANYCPU // implies .net core and STEAMWORKS_X86
+		private static void BeAnyCPUCompatible() {
+			if (s_isAnyCPUHookApplied)
+				return;
+			
+			s_isAnyCPUHookApplied = true;
+			System.Runtime.Loader.AssemblyLoadContext.Default.ResolvingUnmanagedDll += SteamNativeResolveHook;
+		}
+
+		private static bool s_isAnyCPUHookApplied = false;
+
+		private static nint SteamNativeResolveHook(System.Reflection.Assembly requestAsm, string name) {
+			// check who is requesting steam native. only do loading tricks when we requesting steam native
+			if (requestAsm.GetName().Name != "Steamworks.NET") {
+				return 0;
+			}
+
+			// check is requesting library name matches steam native
+			if ((name == NativeMethods.NativeLibraryName || name == NativeMethods.NativeLibrary_SDKEncryptedAppTicket)
+				// check are we on win64, the special case we are going to handle
+				&& RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && System.Environment.Is64BitProcess) {
+
+				string x64LibName = name + "64.dll";
+				// we don't care failed load, let next handler manage it
+				NativeLibrary.TryLoad(x64LibName, out nint lib);
```

Comment:
This will only work if the 64bit dll is present on the current native search path. It will not call any other `ResolvingUnmanagedDll` callbacks.

You can improve compatibility by:
1. Using `NativeLibrary.TryLoad(x64LibName, assembly, null, out nint lib);`, this is most similar to the default call made when a native library is requested by a managed assembly
2. Using `steam_api64` as the lib name (without the `.dll` suffix). It's not needed for native lib loading, and not passed to `ResolvingUnmanagedDll` in typical use

Reply:
Good suggestion! I was finding way to maintain search order, but nothing out. Note 1 helped a lot.

Update are pushed. It will be packed into a pre-release later.